### PR TITLE
[pt-br] Add pt-br/docs/reference/glossary/kops.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translation for portuguese of the glossary term kops.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/kops.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+  Uma ferramenta CLI que ajuda a criar, destruir, atualizar e manter clusters Kubernetes em nível de produção e altamente disponíveis.
+
+<!--more--> 
+
+{{< note >}}
+O kops tem suporte de disponibilidade geral apenas para AWS. O suporte para o uso do kops com GCE e VMware vSphere está em fase alfa.
+{{< /note >}}
+
+`kops` forcene seu cluster com&#58;
```
Comment (edsoncelio) [inline content/pt-br/docs/reference/glossary/kops.md:22]: typo: `forcene`

File: content/pt-br/docs/reference/glossary/kops.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+  Uma ferramenta CLI que ajuda a criar, destruir, atualizar e manter clusters Kubernetes em nível de produção e altamente disponíveis.
+
+<!--more--> 
+
+{{< note >}}
+O kops tem suporte de disponibilidade geral apenas para AWS. O suporte para o uso do kops com GCE e VMware vSphere está em fase alfa.
+{{< /note >}}
+
+`kops` fornece seu cluster com&#58;
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/kops.md:22]: Suggestion: "fornece as seguintes funcionalidades ao seu cluster".

File: content/pt-br/docs/reference/glossary/kops.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  * Instalação totalmente automatizada
+  * Identificação de cluster baseada em DNS
+  * Auto correção&#58; tudo é executado em grupos de Auto-Scaling
+  * Suporte limitado ao sistema operacional (preferência Debian, suportado Ubuntu 16.04, suporte inicial para CentOS & RHEL)
+  * Suporte de alta disponibilidade (_High-Availability_, HA)
+  * A capacidade de provisionar diretamente ou gerar manifestos Terraform
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/kops.md:29]: I would remove the article "A" in this sentence.

**Review by stormqueen1990 — COMMENTED** [review]
Hi there, @MrErlison! 👋

Thank you very much for your contribution. This glossary entry is looking very different from the current live English version, could you please take a look and check if it has changed since you translated it?

**Comment by MrErlison** [comment]
> Hi there, @MrErlison! 👋
> 
> Thank you very much for your contribution. This glossary entry is looking very different from the current live English version, could you please take a look and check if it has changed since you translated it?

Sorry. My main branch was out of date. I fixed.

File: content/pt-br/docs/reference/glossary/kops.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+full_link: /pt-br/docs/getting-started-guides/kops/
+short_description: >
+  kOps não só te ajudará a criar, destruir, atualizar e manter clusters Kubernetes em nível de produção e altamente disponíveis, mas também fornecerá a infraestrutura de nuvem necessária.
+
+aka: 
+tags:
+- tool
+- operation
+---
+  `kOps` não só te ajudará a criar, destruir, atualizar e manter clusters Kubernetes em nível de produção e altamente disponíveis, mas também fornecerá a infraestrutura de nuvem necessária.
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/glossary/kops.md:14]: nit: I think we could use "provisionará" instead of "fornecerá" (compare provision to provides).

**Review by stormqueen1990 — COMMENTED** [review]
Hi there, @MrErlison!

I have a minor nit. Other than that, LGTM!

Thank you very much for your contribution! 🎉

**Comment by edsoncelio** [comment]
Thanks! looks good to me
/lgtm

**Review by stormqueen1990 — COMMENTED** [review]
Thank you very much for your contribution @MrErlison! 😃 

/approve

**Pull request merged**